core/src/main/scala/org/apache/spark/partial/SumEvaluator.scala

```diff
@@ -56,9 +56,12 @@ private[spark] class SumEvaluator(totalOutputs: Int, confidence: Double)
       val confFactor = {
         if (counter.count > 100) {
           new NormalDistribution().inverseCumulativeProbability(1 - (1 - confidence) / 2)
-        } else {
+        } else if (counter.count > 1) {
           val degreesOfFreedom = (counter.count - 1).toInt
           new TDistribution(degreesOfFreedom).inverseCumulativeProbability(1 - (1 - confidence) / 2)
+        } else {
+          // this may not be statistically meaningful
+          confidence
```

Comment:
This isn't valid; confidence is really a probability and confFactor is a number of standard deviations.